subworkflows/local/bam_joint_calling_germline_sentieon/main.nf

```diff
@@ -0,0 +1,115 @@
+//
+include { BCFTOOLS_SORT                                                      } from '../../../modules/nf-core/bcftools/sort/main'
+include { SENTIEON_APPLYVARCAL as SENTIEON_APPLYVARCAL_INDEL                 } from '../../../modules/nf-core/sentieon/applyvarcal/main'
+include { SENTIEON_APPLYVARCAL as SENTIEON_APPLYVARCAL_SNP                   } from '../../../modules/nf-core/sentieon/applyvarcal/main'
+include { SENTIEON_GVCFTYPER                                                 } from '../../../modules/nf-core/sentieon/gvcftyper/main'
+include { GATK4_MERGEVCFS as MERGE_GENOTYPEGVCFS                             } from '../../../modules/nf-core/gatk4/mergevcfs/main'
+include { GATK4_MERGEVCFS as MERGE_VQSR                                      } from '../../../modules/nf-core/gatk4/mergevcfs/main'
+include { SENTIEON_VARCAL as SENTIEON_VARCAL_INDEL                           } from '../../../modules/nf-core/sentieon/varcal/main'
+include { SENTIEON_VARCAL as SENTIEON_VARCAL_SNP                             } from '../../../modules/nf-core/sentieon/varcal/main'
+
+workflow BAM_JOINT_CALLING_GERMLINE_SENTIEON {
+    take:
+    input                // channel: [ meta, [ input ], [ input_index ], intervals ]
+    fasta                // channel: [ fasta ]
+    fai                  // channel: [ fasta_fai ]
+    dict                 // channel: [ dict ]
+    dbsnp
+    dbsnp_tbi
+    dbsnp_vqsr
+    resource_indels_vcf
+    resource_indels_tbi
+    known_indels_vqsr
+    resource_snps_vcf
+    resource_snps_tbi
+    known_snps_vqsr
+
+    main:
+    versions = Channel.empty()
+
+    sentieon_input = input
+        .map{ meta, gvcf, tbi, intervals -> [ [ id:'joint_variant_calling', intervals_name:intervals.simpleName, num_intervals:meta.num_intervals ], gvcf, tbi, intervals ] }
+        .groupTuple(by:[0, 3])
+
+    SENTIEON_GVCFTYPER(sentieon_input, fasta, fai, dbsnp, dbsnp_tbi)
+
+    // TO-DO: Try to use a much functionality from Sentieon as possible. Sentieon should have a function for merging vfcs.
```

Comment:
should make an issue of this, or will be forgotten

Reply:
Removed the comment.